Tests/SKCoreTests/BuildSystemManagerTests.swift

```diff
@@ -429,7 +429,7 @@ class ManualBuildSystem: BuildSystem {
 
   var map: [DocumentURI: FileBuildSettings] = [:]
 
-  var delegate: BuildSystemDelegate? = nil
+  weak var delegate: BuildSystemDelegate? = nil
```

Comment:
`Tests/SourceKitLSPTests/BuildSystemTests.swift` has one too 😅. Can't help but think we're going to continue having this issue.

Reply:
But that’s already `weak`. Or am I missing something?

https://github.com/apple/sourcekit-lsp/blob/1fe79781dfd7ca7024b0f6ccd87ef759fe133870/Tests/SourceKitLSPTests/BuildSystemTests.swift#L30

Reply:
Only that it was late and I assume I read `weak` as `public` or something?